test/language/templates/template-values/line-continuation.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 11.8.6
+description: >
+    The TRV of LineContinuation :: \ LineTerminatorSequence is the sequence
+    consisting of the code unit value 0x005C followed by the code units of TRV
+    of LineTerminatorSequence.
+---*/
+var calls;
+
+calls = 0;
+(function(cs) {
+  calls++;
+  assert.sameValue(cs.raw[0], '\u005C\n\u005C\n\u005C\n');
+})`\
+\
+\
`
```

Comment:
I think this template literal should also test <LS>, <PS>, and <CR><LF> (similar to line-terminator-sequence?)